draft-ietf-ppm-dap.md

```diff
@@ -2436,195 +2436,55 @@ suite:
-1. Relaying messages between Helper and Collector in the collect sub-protocol.
-   These messages are not authenticated, meaning the leader can:
-   1. Send collect parameters to the Helper that do not reflect the parameters
-      chosen by the Collector
-      * This is mitigated by including the `BatchSelector` and aggregation
-        parameter in the AAD used to encrypt aggregate shares.
-   1. Discard the aggregate share computed by the Helper and then fabricate
-      aggregate shares that combine into an arbitrary aggregate result
-   * These are attacks on robustness, which we already assume to hold only if
-     both Aggregators are honest, putting these malicious-Leader attacks out of
-     scope.
+1. Clients may affect the quality of aggregate results by reporting false
+   measurements. A VDAF can only verify that a submitted measurement is valid,
+   not that it is true.
+1. Clients can intentionally leak their own measurements and compromise their
+   own privacy.
+1. An attacker can impersonate multiple Clients, or a single malicious Client
+   can upload an unexpectedly-large number of reports, in order to skew
+   aggregate results or to reduce the number of measurements from honest Clients
+   in a batch below the minimum batch size. See {{sybil}} for discussion and
+   potential mitigations.
+1. Both Aggregators can collude to share unencrypted input shares in order to
+   defeat the privacy of individual reports. DAP follows VDAF in providing
+   privacy only if at least one Aggregator is honest.
+1. Aggregators can defeat robustness by emitting incorrect aggregate shares, by
+   omitting reports from the aggregation process, or by forging a proof that
+   will fool honest Aggregators into accepting an invalid report. DAP follows
+   VDAF in providing robustness only if both Aggregators are honest.
+1. Aggregators, or a passive network attacker between the Clients and the
+   Leader, can examine metadata such as client IP in order to infer which
+   Clients are submitting reports. This may be used to infer sensitive
+   information about the Client. This can be mitigated for the Aggregator by
+   deploying an anonymizing proxy (see {{anon-proxy}}), or in general by
+   requiring Clients to submit reports at regular intervals independently of the
+   measurement value such that the existence of a report does not imply the
+   occurrence of a sensitive event.
+1. Aggregators can deny service by refusing to respond to collection requests or
+   aggregate share requests.
+1. Collection requests may leak information beyond the collection results. For
+   example, the Poplar1 VDAF {{!VDAF}} can be used to compute the set of heavy
+   hitters among a set of arbitrary bit strings uploaded by Clients. This
+   requires multiple evaluations of the VDAF, the results of which reveal
+   information to the Aggregators and Collector beyond what follows from the
+   heavy hitters themselves. Or the result of the Prio3Sum VDAF could leak
```

Comment:
Not specific to any VDAF. I think Prio3Sum is meant only as an example, but calling it out specifically may make a casual reader think it's more vulnerable than other VDAFs.
```suggestion
   heavy hitters themselves. Or the result could leak
```